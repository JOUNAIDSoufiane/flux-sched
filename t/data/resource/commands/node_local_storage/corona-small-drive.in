match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/corona-small-drive.yaml
match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/corona-small-drive.yaml
quit
