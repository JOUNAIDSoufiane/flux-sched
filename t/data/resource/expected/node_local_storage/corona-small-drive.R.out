corona-full-drive.R.out